solidity/contracts/MailboxV3.sol

```diff
@@ -182,4 +192,34 @@ contract MailboxV3 is IMailboxV3, Versioned, Ownable {
         } catch {}
         return defaultIsm;
     }
+
+    // ============ Internal Functions ============
+
+    function _dispatch(
```

Comment:
why separate this from the above?

Reply:
nit: preference for unified interface between the overloaded functions